native/cocos/renderer/gfx-metal/MTLDevice.mm

```diff
@@ -162,6 +161,10 @@ of this software and associated engine source code (the "Software"), a limited,
 }
 
 void CCMTLDevice::doDestroy() {
+    // force wait done
+    while(_inFlightCount.load(std::memory_order_acquire) != 0) {
+        ;
```

Comment:
Will `while` cause high cpu consumption? We might sleep for some time here.

Reply:
Nice point, adding.